frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java

```diff
@@ -157,6 +157,21 @@ public void testDescribeApi() throws InterruptedException {
     @Test(
             alwaysRun = true,
             dependsOnMethods = {"testDescribeApi"})
+    public void testInitialWorkers() throws InterruptedException {
+        Channel channel = TestUtils.getManagementChannel(configManager);
+        TestUtils.setResult(null);
+        TestUtils.setLatch(new CountDownLatch(1));
+        TestUtils.describeModel(channel, "noop", null);
+        TestUtils.getLatch().await();
+        DescribeModelResponse[] resp =
+                JsonUtils.GSON.fromJson(TestUtils.getResult(), DescribeModelResponse[].class);
+        Assert.assertEquals(TestUtils.getHttpStatus(), HttpResponseStatus.OK);
+        Assert.assertEquals(resp[0].getMinWorkers(), configManager.getDefaultWorkers());
+    }
+
+    @Test(
+            alwaysRun = true,
+            dependsOnMethods = {"testInitialWorkers"})
```

Comment:
Add UT to validate the workers for model registered via api [with/without initial workers.]

Reply:
Done.